Library/Homebrew/cask/audit.rb

```diff
@@ -487,11 +491,14 @@ def audit_signing
 
           next if result.success?
 
-          add_error <<~EOS, location: cask.url.location, strict_only: true
+          add_error <<~EOS, location: cask.url.location
```

Comment:
Removing the `strict_only: true` parameter will cause this audit to be applied to all cask audits, not just when the `--new-cask` flag is passed to `brew audit`. This should be considered separately.